main.py

```diff
@@ -162,3 +176,62 @@ def _execute_cmd_and_parse_version_from_output(cmd):
     if match_object is None:
         raise Exception("Cannot parse version number from ouput: '{0}'".format(output))
     return match_object.groups()[0]
+
+def _init_telemetry(settings):
```

Comment:
Can the telemetry related functions be moved to a separate file, maybe `/libs/telemetry.py`? It would be cleaner and easier to track changes about telemetry.